torch/csrc/autograd/python_cpp_function.cpp

```diff
@@ -201,6 +201,13 @@ PyObject* THPCppFunction_sequence_nr(PyObject* self, PyObject* noargs) {
   auto& fn = *((THPCppFunction*)self)->cdata;
   return THPUtils_packUInt64(fn.sequence_nr());
 }
+
+PyObject* THPCppFunction_set_sequence_nr(PyObject* self, PyObject* sequence_nr) {
+  auto& fn = *((THPCppFunction*)self)->cdata;
+  fn.set_sequence_nr(THPUtils_unpackUInt64(sequence_nr));
+  Py_RETURN_NONE;
```

Comment:
```suggestion
  HANDLE_TH_ERRORS
  auto& fn = *((THPCppFunction*)self)->cdata;
  fn.set_sequence_nr(THPUtils_unpackUInt64(sequence_nr));
  Py_RETURN_NONE;
  END_HANDLE_TH_ERRORS
```

Reply:
I didn't see any of the other methods in cpp_function having this (but I did have it in python_function.cpp)

Reply:
because most of them cannot raise c++ errors and handle python errors by hand (for historical reasons, these macro might not have existed when this code was written).
THPUtils_unpackUInt64 will throw a nice c++ error on error which you need to handle here.

Reply:
ahh gotcha, thanks